exporter/opentelemetry-exporter-otlp-proto-grpc/tests/logs/test_otlp_logs_exporter.py

```diff
@@ -212,6 +214,10 @@ def test_exporting(self):
             OTEL_EXPORTER_OTLP_LOGS_ENDPOINT: "logs:4317",
             OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE: THIS_DIR
             + "/../fixtures/test.cert",
+            OTEL_EXPORTER_OTLP_LOGS_CLIENT_CERTIFICATE: THIS_DIR
+            + "/../fixtures/test-client-cert.pem",
+            OTEL_EXPORTER_OTLP_LOGS_CLIENT_KEY: THIS_DIR
```

Comment:
Wouldn't it be better to leave this test as-is and create a new one with the client_certificate and client_key?

Reply:
@emdneto thanks I will modify it, and I am wondering if there is a good suggestion to overcome the 1000-Line per file limitation? Should I just separate the test file?

Reply:
Since it's a test I'm fine with `# pylint: disable=too-many-lines`

Reply:
Resolved, tyvm!